discord/channel.py

```diff
@@ -120,6 +121,50 @@
 
     OverwriteKeyT = TypeVar('OverwriteKeyT', Role, BaseUser, Object, Union[Role, Member, Object])
 
+    class _CreateChannelWithCategory(TypedDict, total=False):
+        category: Optional[CategoryChannel]
+
+    class _CreateNewsChannel(TypedDict, total=False):
+        news: bool
+
```

Comment:
These appear unused.

Reply:
These were intended for future use, but now that I think about it, that seems a bit silly. I've removed them.